### DISCUSSION TITLE
a specific case question

### DISCUSSION THREAD

**Discussion by gen3111620** [milvus-io/milvus#28038] [Q&A and General discussion] [answered]
If I have a situation today where I need to use IVF_PQ to quickly filter out K pieces of data from billions of records and then employ a brute-force method to produce the final set of M data from the remaining K.

How should I appropriately proceed in Milvus?

ref : https://zhuanlan.zhihu.com/p/378725270

> 业务方通常会使用IVFPQ加brute-force。比如用IVFPQ粗略地找出Top K条，然后再在这K条中进行暴力计算，找出Top M条作为最终结果

**Comment by yhmo** [comment, marked as answer]
Good case. Currently, milvus doesn't support multiple indexes on the same vector field, you are not able to search with different indexes unless you recreate the index. Another workaround is: create two collections with the same data, one is IVF_PQ index, the other is FLAT index, but this approach wastes memory.
